eth/ethconfig/config.go

```diff
@@ -270,6 +270,7 @@ var ChainsWithSnapshots = map[string]struct{}{
 	networkname.BorMainnetChainName: {},
 	networkname.GnosisChainName:     {},
 	networkname.ChiadoChainName:     {},
+	// networkname.ClassicChainName:    {},
```

Comment:
Does Classic have snapshots that are compatible with Erigon? 
Classic has snapshots on the network per the ethereum/go-ethereum protocol, but not sure this applies here.